dao/elasticsearch/makefile

```diff
@@ -18,8 +18,5 @@ default: build
 build:
 	go build
 
-test: build
-	go test $(GOTEST_FLAGS)
-
 clean:
```

Comment:
Remove recursive make testing scheme in favor of go test ./... recursive testing scheme.